recipes/units/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from conans import CMake, ConanFile
+
+
+class UnitsTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        os.chdir("bin")
+        self.run(".%sunitstest" % os.sep)
```

Comment:
```suggestion
        if not tools.cross_building(self.settings):
            bin_path = os.path.join("bin", "unitstest")
            self.run(bin_path, run_environment=True)
```
Support cross building

Reply:
Thanks, this change is in.